queue_job_chunk/components/processor.py

```diff
@@ -0,0 +1,11 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from odoo.addons.component.core import AbstractComponent
+
+
+# DISCUSSION: abstractcomponent?
+class Processor(AbstractComponent):
+    _name = "processor"
+    _inherit = "base"
+    _collection = "collection.base"
```

Comment:
```suggestion
    _collection = "queue.job.chunk"
```

Reply:
https://github.com/OCA/connector/blob/ec96b99940ecf77a94623c226f0a52833c447deb/component/models/collection.py#L96

According to this, every time we would use collection.work_on (...) we would need to create a dummy queue.job.chunk, which seems impractical since we have required fields

Reply:
in fact when we call the run method we called it on a existing queue.job.chunk so it should always work